plugins/hydra_rq_launcher/news/1586.plugin

```diff
@@ -0,0 +1 @@
+Added support for SSL Redis connections in the RQ launcher plugin.
```

Comment:
Complete the sentence "This release will ..."
```suggestion
Add support for SSL Redis connections in the RQ launcher plugin.
```

Reply:
✅